src/modules/saveReminder/SaveReminder.ts

```diff
@@ -0,0 +1,60 @@
+import {
+    Observable as KnockoutObservable,
+} from 'knockout';
+import { Saveable } from '../DataStore/common/Saveable';
+import * as GameConstants from '../GameConstants';
+import Notifier from '../notifications/Notifier';
+import Settings from '../settings';
+
+export default class SaveReminder implements Saveable {
+    public static counter = 0;
```

Comment:
counter looks unused

Reply:
The counter is incremented within `Game.ts`, which runs the `tick` function every 5 minutes.

Reply:
Yes, but we don't use the value anywhere, right?

Reply:
The value is also used in `Game.ts`
```ts
        SaveReminder.counter += GameConstants.TICK_TIME;
        if (SaveReminder.counter >= 5 * GameConstants.MINUTE) {
```

Reply:
Missed that.
Great! Will look at the other change later today